Bottom: 8ab9374ed9c72f1156704013441fc276bfc2404f
Top:    ea161c78b16e178a00f6817cce3e6c14fc5b5237
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-24 12:40:50 +0100

Refresh of fixup-gtk-hid-drawing-routines

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 56e0a8d..9159eed 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1273,7 +1273,7 @@ pour_callback_no_clear (const BoxType * b, void *cl)
   struct poly_info *i = (struct poly_info *) cl;
   PourType *pour = (PourType *)b;
 
-  DrawPour (i->Layer, pour, 0);
+  DrawPour (i->Layer, pour);
 
   if (pour->PolygonN)
     {
@@ -1289,7 +1289,7 @@ pour_callback_clearing (const BoxType * b, void *cl)
   struct poly_info *i = (struct poly_info *) cl;
   PourType *pour = (PourType *)b;
 
-  DrawPour (i->Layer, pour, 0);
+  DrawPour (i->Layer, pour);
 
   if (pour->PolygonN)
     {
